gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/desktop/application/dsl/PlatformSettings.kt

```diff
@@ -36,6 +36,8 @@ abstract class AbstractMacOSPlatformSettings : AbstractPlatformSettings() {
     var appCategory: String? = null
     var minimumSystemVersion: String? = null
 
+    var dmgContents = ArrayList<String>()
```

Comment:
since this is supported from 21 onwards, I considered having this option opt-in to make sure people are aware that this only works when building with jdk 21

But possibly I am overthinking this, what do you think?

Reply:
Is it possible to check the toolchain here, and issue a warning (and ignore the property) if current JDK is older than 21?

Reply:
it probably is possible, but I would need a pointer to somewhere this has been done to make sure I get it right

Reply:
I found `if (jvmRuntimeInfo.majorVersion >= 18) { ` on my own 😄